packages/desktop-client/src/components/common/Modal.tsx

```diff
@@ -418,6 +419,7 @@ export function ModalTitle({
       {shrinkOnOverflow ? (
         <AutoTextSize
           as={Text}
+          aria-label="Modal title"
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 warning:** ‏we should not be setting aria labels to elements that already have perfectly legible titles (in this case `title`). Instead you might consider using roles: `aria-role="heading"` and then lookup via `getByRole('heading')`